src/plugins/condition/components/TestData.vue

```diff
@@ -31,10 +31,10 @@
       <div class="c-cs__header-label c-section__label">Test Data</div>
     </div>
     <div v-if="expanded" class="c-cs__content">
-      <div class="c-cs__test-data__controls c-cdef__controls" :disabled="!telemetry.length">
+      <div :class="testDataClass">
```

Comment:
I think an [array syntax](https://v3.ru.vuejs.org/guide/class-and-style.html#array-syntax) here would be cleaner.